data/mods/Aftershock/items/inactiverobot.json

```diff
@@ -191,5 +193,31 @@
     "symbol": ";",
     "color": "green",
     "use_action": { "type": "place_monster", "monster_id": "afs_mon_sentinel_lx", "difficulty": 4, "moves": 100, "skill2": "computer" }
+  },
+  {
+    "id": "bot_bloodhound_drone",
+    "type": "TOOL",
+    "name": "inactive bloodhound drone",
+    "description": "This is an inactive bloodhound drone.  An automated seeker drone designed to harry and harrass while maintaining visual contact for other pursuers, a small spiked quadcopter robot with a high powered spotlight.",
```

Comment:
```suggestion
    "description": "This is an inactive bloodhound drone.  If successfully deployed, it will fly towards enemy targets and highlight them with a powerful spotlight.",
```